rllib/agents/a3c/a2c_workflow.py

```diff
@@ -0,0 +1,33 @@
+"""Experimental workflow-based impl; run this with --run='A2C_wf'"""
+
+import math
+
+from ray.rllib.agents.a3c.a2c import A2CTrainer
+from ray.rllib.utils.experimental_dsl import (
+    ParallelRollouts, ConcatBatches, ComputeGradients, AverageGradients,
+    ApplyGradients, TrainOneStep, StandardMetricsReporting)
+
+
+def training_workflow(workers, config):
+    rollouts = ParallelRollouts(workers, mode="bulk_sync")
+
+    if config["microbatch_size"]:
+        num_microbatches = math.ceil(
+            config["train_batch_size"] / config["microbatch_size"])
+
+        train_op = rollouts \
+            .combine(ConcatBatches(min_batch_size=config["microbatch_size"])) \
+            .for_each(ComputeGradients(workers)) \
+            .batch(num_microbatches) \
+            .for_each(AverageGradients()) \
+            .for_each(ApplyGradients(workers))
+    else:
+        train_op = rollouts \
+            .combine(ConcatBatches(
+                min_batch_size=config["train_batch_size"])) \
+            .for_each(TrainOneStep(workers))
```

Comment:
It's hard to know what operators are operating on "Batches", "Gradients", and "Workers".